root.sh

```diff
@@ -97,6 +97,15 @@ else
   ROOT_HAS_NO_PYTHON=1
 fi
 
+if [ -n "$XROOTD_ROOT" ]; then
```

Comment:
@TimoWilken @ktf  Why shouldn't we use just a plain `-Dbuiltin_xrootd=0` ? This way, the xrootd capability of root is directly dependent on presence of xrootd itself in the system...

Reply:
which we do not control and which could give issues, though...